### PR TITLE
[stable/openebs] add deprecation notice to openebs chart

### PR DESCRIPTION
**Pull request by kmova**
Signed-off-by: kmova <kiran.mova@mayadata.io>

#### Is this a new chart

No. 

> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please host your own chart repository and submit your repository to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new chart repositories to the Helm Hub.

#### What this PR does / why we need it:

Add deprecation notice. The  new location for the chart is at https://github.com/openebs/charts
The new chart location has been updated at: https://hub.helm.sh/charts?q=openebs

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - Ref #  https://github.com/helm/charts/issues/21103

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/openebs/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-version: 1.11.0
+version: 1.11.1
 name: openebs
```
Comment (davidkarlsen) [inline stable/openebs/Chart.yaml:3]:
```suggestion
name: openebs
deprecated: true
```

File: stable/openebs/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 appVersion: 1.11.0
-description: Containerized Storage for Containers
+description: DEPRECATED Containerized Storage for Containers
```
Comment (davidkarlsen) [inline stable/openebs/Chart.yaml:7]:
```suggestion
# The chart is deprecated and moved. See README.md
deprecated: true
description: DEPRECATED Containerized Storage for Containers
```

Comment (kmova) [reply stable/openebs/Chart.yaml:7]: Thank you for catching this!

**Pull request merged**